tabpfn_client/client.py

```diff
@@ -157,15 +158,20 @@ class ServiceClient(Singleton):
     httpx_timeout_s = (
         4 * 5 * 60 + 15  # temporary workaround for slow computation on server side
     )
-    httpx_client = httpx.Client(
+    httpx_client = AnalyticsHttpClient(
         base_url=base_url,
         timeout=httpx_timeout_s,
         headers={"client-version": get_client_version()},
+        module_name="tabpfn_client",
```

Comment:
Are you sure there is no way of getting the module name programmatically?

Reply:
At least not robustly, in my opinion.
My rationale: `tabpfn-extensions` or wrappers built by users can create arbitrarily deep call stack. So I thought it'd be easier to let the wrapper, e.g. `tabpfn-extensions`, set the module name explicitly.

Reply:
I see. I was hoping there is a better way to get the module name than iterating the call stack.